devices/rpi-b/raspberrypios-buster-armhf/README.md

```diff
@@ -1,4 +1,4 @@
-Raspbian is an official operating system for Raspberry Pi. It's based on Debian GNU/Linux and supports all models of the single-board computer. `rpi-b/raspbian-buster-armhf` allows building custom 32-bit Raspbian [Buster](https://raspberrypi.org/blog/buster-the-new-version-of-raspbian/) images for
+Raspberry Pi OS is an official operating system for Raspberry Pi. It's based on Debian GNU/Linux and supports all models of the single-board computer. `rpi-b/raspberrypios-buster-armhf` allows building custom 32-bit Raspberry Pi OS (formerly Raspbian) [Buster](https://raspberrypi.org/blog/buster-the-new-version-of-raspbian/) images for
```

Comment:
No, move `(formerly Raspbian)` to the beginning of the sentence like `Raspberry Pi OS (formerly Raspbian) is...`